src/v3/test/integration/admin-consent.test.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { createAuthJsPayloadArgs, setup } from './util';
+
+import adminConsentResponse from '../../../../playground/mocks/data/idp/idx/consent-admin.json';
+
+describe('admin-consent', () => {
+  it('should render form with logo', async () => {
+    const { container, findByRole } = await setup({ mockResponse: adminConsentResponse });
+    const appNameHeading = await findByRole('heading', { level: 2 });
+    const groupHeading = await findByRole('heading', { level: 3 });
+
+    expect(appNameHeading.textContent).toBe('Native client');
+    expect(groupHeading.textContent).toBe('Resource and policies');
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render form without logo', async () => {
+    const adminConsentResponseWithoutLogo = {
+      ...adminConsentResponse,
+      app: {
+        ...adminConsentResponse.app,
+        value: { ...adminConsentResponse.app.value, logo: undefined },
+      },
+    };
+    const { container, findByRole } = await setup({
+      mockResponse: adminConsentResponseWithoutLogo,
+    });
+    const appNameHeading = await findByRole('heading', { level: 2 });
+    const groupHeading = await findByRole('heading', { level: 3 });
+
+    expect(appNameHeading.textContent).toBe('Native client');
+    expect(groupHeading.textContent).toBe('Resource and policies');
+
+    expect(container).toMatchSnapshot();
```

Comment:
is there an assertion to check if the logo is present or not?

Reply:
No assertion, I was hoping we could rely on the snapshot to ensure it does not regress. I can add an assertion though to be extra sure.

Reply:
okay yeah I think relying on the snapshot is fine, but if you could add an assertion to be extra sure that would be good